doc/nrf/releases/release-notes-latest.rst

```diff
@@ -12,19 +12,34 @@ The most relevant changes that are present on the master branch of the |NCS|, as
 .. note::
     This file is a work in progress and might not cover all relevant changes.
 
-* ``bl_boot`` library:
-
-    * Disable clock interrupts before booting app.
-      This fixes an issue where the :ref:`bootloader` sample would not be able to boot a zephyr application on the nRF5340.
-
 Changelog
 *********
 
-See the following sections for a list of the most important changes.
+The following sections provide detailed lists of changes by component.
+
+nRF5
+====
+
+The following changes are relevant for the nRF52 and nRF53 Series.
+
+nRF5340 SoC
+-----------
+
+* Updated:
 
+  * ``bl_boot`` library - Disabled clock interrupts before booting app.
+    This change fixes an issue where the :ref:`bootloader` sample would not be able to boot a zephyr application on the nRF5340 SoC.
```

Comment:
@hakonfam , is this the correct placement for this entry?

Reply:
Looks better than the first placement!

To me this seems good :)